### PR TITLE
Fixed start location popup to not linger after information is entered

### LINKED ISSUE #72: Start location popup remains open after information is entered
**Steps to reproduce:**
- Click the Navigate button at the bottom left
- Click "Search for location"
- Enter a start and end location
- Press OK

**Expected:**
- All popups close and the route is displayed

**Actual:**
- The first prompt (to select the method of specifying the start location) remains open, concealing the map with the route.

**Workaround:**
- Tap outside of the prompt window to close it

### PR DESCRIPTION
**Pull request by jonZlotnik**
another one liner...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,11 +22,9 @@ Create a campus guide application to help students locate their classrooms.
...
-6. The file contains a single `<string>` tag. Change the `name` property from `google_maps_key_ex` to `google_maps_key`. Change the contents of the tag from `google_maps_key` to the value of your API key (it starts with `AIza`)
-7. Perform steps 4 through 6 with the file `app\src\release\res\values\google_maps_api.example.xml`. This step is only necessary to build the app in Release configuration.
-8. Open the project in Android Studio
-9. Run the project using an emulator or using a connected android device.
+1. [Get an API key for Google Maps](https://developers.google.com/maps/documentation/android-sdk/get-api-key)
+1. Clone the repository onto your computer
+1. Create a file `app\src\main\res\values\google_maps_api.xml`.
+1. Write the following to the file: `<resources><string name="google_maps_key">*API_KEY_GOES_HERE*</string></resources>`
+1. Open the project in Android Studio
+1. Run the project using an emulator or using a connected android device.
```
Comment (MikeBrichko) [inline README.md:30]: All the steps have "1.", they should probably be numbered as it was previously 1-2-3-4-5..etc

File: app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptionsDialogFragment.kt [hunk 2]
```diff
@@ -20,7 +20,7 @@ import com.google.android.gms.location.LocationServices
 import com.google.android.gms.tasks.OnCompleteListener
 
 
-class ChooseDirectionOptions constructor(private val route: Route?) : DialogFragment() {
+class ChooseDirectionOptionsDialogFragment constructor(private val route: Route?) : DialogFragment() {
```
Comment (MikeBrichko) [inline app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptionsDialogFragment.kt]: Is it necessary to add "DialogFragment" in the class name if it already extends DialogFragment()? I'm not sure what the best practice is in this case

**Review by MikeBrichko — DISMISSED** [review]
Let me know what you think of my review

File: README.md (hunk 1 above)
Comment (felixlapierre) [reply README.md]: This is part of another pull request.

Comment (felixlapierre) [reply README.md:30]: This is part of another pull request

File: app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptionsDialogFragment.kt (hunk 2 above)
Comment (felixlapierre) [reply app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptionsDialogFragment.kt]: IMO it should at least have Dialog, and i'd be okay with DialogFragment

Why? ChooseDirectionOptions sounds like a data transfer object containing the options available when choosing directions. That is not what this class does at all: it's a UI component, and having DialogFragment, or at least Dialog, conveys that information well.

File: app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptionsDialogFragment.kt [hunk 3]
```diff
@@ -92,6 +92,7 @@ class ChooseDirectionOptions constructor(private val route: Route?) : DialogFrag
                         CallbackDirectionsConfirmListener { start, end ->
                             //Display the directions time
                             route?.set(start, end)
+                            this.dismiss()
```
Comment (felixlapierre) [inline app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptionsDialogFragment.kt]: 🤦‍♂ so simple!

**Review by felixlapierre — APPROVED** [review]
Looks good to me! 😄

File: app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptionsDialogFragment.kt (hunk 3 above)
Comment (jonZlotnik) [reply app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptionsDialogFragment.kt]: Haha, yeah. Just needed to track down where it needed to go.

**Pull request merged**